benchmarks/2021.json

```diff
@@ -49,14 +49,14 @@
     "isToppedOutByProgram": false,
     "deciles": [
       100,
-      90.69,
-      72.51,
-      55.17,
-      41.98,
-      32.56,
-      25.48,
-      19.15,
-      12.87
+      79.99,
+      70,
+      60,
+      50,
+      40,
+      30,
+      20,
+      10
```

Comment:
Are these all exactly multiples of ten, except one, when last year they were all to two decimal places?

Reply:
this one jumped out as well, it's only happening with 001 too...

Reply:
This is because 001 is a flat benchmark. I'd expect to see this behavior for 236 as well.

Reply:
Yes you are correct! I forgot about the Flats!